launch.go

```diff
@@ -149,7 +172,7 @@ func launch(api screwdriver.API, buildID int, rootDir, emitterPath string) error
 	}
 
 	log.Print("Setting Build Status to RUNNING")
-	if err = api.UpdateBuildStatus(screwdriver.Running, buildID); err != nil {
+	if err = api.UpdateBuildStatus(screwdriver.Running, nil, buildID); err != nil {
```

Comment:
@tk3fftk this will fail the validation when hitting the build update api since we expect meta to be an object. 
https://github.com/screwdriver-cd/data-schema/blob/master/models/build.js#L148
Have pasted the log to u in slack